### DISCUSSION TITLE
For user profiles: Social Media Widgets

### DISCUSSION THREAD

**Discussion by digitalwaveride** [Automattic/gravatar#62] [Ideas]
Enriching user profiles to make them more relevant as the main homebase on the web.
 
Could my Gravatar profile be a place to showcase my latest posts across various social media feeds? It would serve as a central hub, providing an overview of where I'm active and teasers of my latest content.

---- 
Feature suggested during user interview

**Comment by aaronfc** [comment]
This was briefly discussed in our p8Slzc-2Ip-p2:
> Aggregated user feeds
>   Combine everyone’s verified service feeds into one, ActivityPub-style feed.
